pcsx2/gui/FrameForGS.cpp

```diff
@@ -557,6 +557,7 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 #endif
 
 	double fps = wxGetApp().FpsManager.GetFramerate();
+	float per = gsRegionMode == Region_NTSC ? (fps * 100) / 59.94 : (fps * 100) / 50; 
```

Comment:
59.94 and 50 should not be hardcoded. They're a GS configuration values (you can see them grayed out at the GS tan of the config panel).